src/LaraCart.php

```diff
@@ -483,16 +486,20 @@ public function total($format = true, $withDiscount = true)
      *
      * @param boolean $format
      * @param boolean $withDiscount
+     * @param boolean $withTax
      *
      * @return string
      */
-    public function subTotal($format = true, $withDiscount = true)
+    public function subTotal($format = true, $withDiscount = true, $withTax = false)
```

Comment:
The reasoning behind sub total cant have taxes is : 

if you have a coupon. You cannot apply the taxes till AFTER you get the sub total.

So this would cause issues of getting the real sub total.